### DISCUSSION TITLE
Duda avanze de Katas

### DISCUSSION THREAD

**Discussion by EAleman8** [LaunchX-InnovaccionVirtual/CursoIntroPython#234] [Q&A] [answered]
Que tal!

Tengo una duda... SÉ que a partir de hoy podemos enviar las katas terminadas y que tendremos hasta el lunes 21 para entregarlas con el enlace en el form.
Pero creo recordar que hubo un en vivo en donde @FernandaOchoa mencionó que se tenía que entregar un avance de lo que lleváramos hecho hasta hoy aunque no estuvieran todas si queríamos conservar nuestra oportunidad a pasar al grupo oficial.

Entonces, debo mandar el enlace de lo que llevo hasta hoy, y después volverlo a mandar cuando las tenga todas terminadas?
O debo esperar a tenerlas todas terminadas y hasta entonces mandar el enlace en el form?

Estoy un poco confundido con eso. 🥲

**Comment by JavierMendez-Coder** [comment, marked as answer]
Buenas noches. Si las estas subiendo todas a un mismo repositorio no creo que sea necesario enviar el enlace nuevamente, ya que, este no cambia aunque lo actualices múltiples veces. 

Espero haber resuelto tu duda. 👋

**Reply by EAleman8** [reply]
De acuerdo.

Muchas gracias!

**Comment by JonnySC** [comment]
Te sugiero contestar el forms que se nos mando correctamente, con tu correo de @innovaccion.mx y tu liga a tu repositorio, y ve actualizando tu repositorio con forme vas terminado cada programa (kata) el propósito de esta misión es medir el conocimiento de cada tripulante :D saludos

**Comment by FernandaOchoa** [comment]
Es correcto, puedes mandar tu avance, al actualizar tu repositorio, conservas la misma liga (enlace) por lo que no hay de qué preocuparse.
